src/server_manager/web_app/ui_components/outline-server-settings.js

```diff
@@ -208,7 +208,7 @@ Polymer({
               <!-- NOTE: The dropdown is not automatically sized to the button's width:
                            https://github.com/PolymerElements/paper-dropdown-menu/issues/229 -->
               <paper-dropdown-menu no-label-float="" horizontal-align="left">
-                <paper-listbox slot="dropdown-content" selected="{{_computeDataLimitsEnabledName(isAccessKeyDataLimitEnabled)}}" attr-for-selected="name" on-selected-changed="_accessKeyDataLimitEnabledChanged">
+                <paper-listbox slot="dropdown-content" selected="{{_computeDataLimitsEnabledName(isDefaultDataLimitEnabled)}}" attr-for-selected="name" on-selected-changed="_defaultDataLimitEnabledChanged">
```

Comment:
For future PRs, it would be better if the renaming to "default data limit" was done in a separate PR.

Reply:
This was my mistake in mismanaging my branches.  https://github.com/Jigsaw-Code/outline-server/pull/771 was only merged into a feature base branch I was attempting to use, but never into master.